src/librustc/session/config.rs

```diff
@@ -1256,6 +1256,9 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         `everybody_loops` (all function bodies replaced with `loop {}`),
         `hir` (the HIR), `hir,identified`, or
         `hir,typed` (HIR with types for each node)."),
+    dep_info_omit_d_target: bool = (false, parse_bool, [TRACKED],
```

Comment:
this looks... like a merge gone wrong? :)

Reply:
@nikomatsakis I'm not sure I understand... This was the merge conflict. Am I missing something?